mutacc/builds/build_variant.py

```diff
@@ -34,28 +34,28 @@ def find_region(self, padding):
             vtype = self.entry.INFO.get("SVTYPE").upper()
 
             if vtype != "BND":
-                
+
                 region = {"start": self.entry.start - padding,
                           "end": self.entry.end + padding}
 
             #Find the region for variant with an 'SVTYPE' ID in the INFO field of the vcf entry
             else:
-                
+
                 #Depending on the direction of the break end, the region is extended either upstreams
                 #or downstream of the position of the break end.
                 if "[" in self.entry.ALT[0]:
```

Comment:
Are the if and else condition defining the same region? Maybe I am missing something?

Reply:
Also, this looks like a very good place to use a technique called a dispatch table. Look it up and see if you can refactor the function.